internal/dash/dash.go

```diff
@@ -282,7 +283,12 @@ func initModuleManager(options *moduleOptions) (*module.Manager, error) {
 
 func buildListener() (net.Listener, error) {
 	listenerAddr := api.ListenerAddr()
-	return net.Listen("tcp", listenerAddr)
+	conn, err := net.DialTimeout("tcp", listenerAddr, time.Second*5)
```

Comment:
Couldn’t the timeout be < than 500ms? 5 seconds would slow dow octant’s start up

Reply:
yeah, I'll make it 500ms that should be plenty of time